content/en/agent/faq/certificate_verify_failed-error.md

```diff
@@ -98,11 +109,33 @@ Datadog recommends keeping up to date and updating to the latest version of the
 
 Yes. The certificate is just a preset for the client to use and is not necessary to connect via SSL. Datadog Agent endpoints only accept SSL traffic.
 
+### Additional step if the Windows Agent 5.x is configured to use a proxy or the curl http client
+
+On Windows, with Agent 5.x, removing the certificate file alone is not enough if the Agent is configured to either:
+
+* use a proxy with the `proxy_host` configuration option in `datadog.conf` or the `HTTPS_PROXY` environment variable, or
+* use the curl http client with the `use_curl_http_client: yes` configuration option in `datadog.conf`
+
+Note: `datadog.conf` is located in `C:\ProgramData\Datadog\datadog.conf`.
+
+In this case, take this additional action:
+
+* Windows Agent v5, `>= 5.12.0`: replace the `datadog-cert.pem` file with the version that is shipped in 5.32.7. Using the Powershell CLI:
+
+```shell
+Invoke-WebRequest -Uri f"https://raw.githubusercontent.com/DataDog/dd-agent/5.32.7/datadog-cert.pem" -OutFile "C:\Program Files\Datadog\Datadog Agent\agent\datadog-cert.pem"
+restart-service -Force datadogagent
+```
+
+* Windows Agent v5, `<= 5.11.x`: set the following option in `datadog.conf`:
+  * 64-bit Windows: `ca_certs: C:\Program Files (x86)\Datadog\Datadog Agent\files\ca-certificates.crt`
+  * 32-bit Windows: `ca_certs: C:\Program Files\Datadog\Datadog Agent\files\ca-certificates.crt`
```

Comment:
related to the comment in the `Using the Windows GUI` section. Do users need to delete datadog-cert.pem too? if so, we should mention that here.

Reply:
(as discussed in other comment thread, the answer is that users do not need to delete the cert first)